solr/core/src/java/org/apache/solr/handler/admin/CollectionsHandler.java

```diff
@@ -590,6 +590,7 @@ public enum CollectionOperation implements CollectionOp {
               CREATE_NODE_SET,
               CREATE_NODE_SET_SHUFFLE,
               "shards",
+              "chkConfigName",
```

Comment:
maybe use a constant similar to lines 584-586 above
```suggestion
              ReindexCollectionCmd.CHK_CONFIG_NAME,
```

Reply:
yeah, thanks!